curvefs/test/mds/chunkid_allocator_test.cpp

```diff
@@ -104,7 +104,7 @@ TEST_F(ChunkIdAllocatorTest, test_get_chunkIds_1001) {
     set<uint64_t> chunkids;  // record get chunkids
     for (int i = 0; i < times; ++i) {
         uint64_t tmp = 0;
-        chunkIdAllocator_->GenChunkId(&tmp);
+        chunkIdAllocator_->GenChunkId(1, &tmp);
```

Comment:
it's better to add test case for allocating more chunks at once.

Reply:
Agree, and preferably some limit value or random value, and judge whether the returned value is correct each time.

Reply:
fixed